gnosis/cowswap
# adjusts for connect a wallet btn

# Summary

Fixes #1817  

![image](https://user-images.githubusercontent.com/622217/141329313-9d8e34b4-11cc-4038-b0ac-b4c9470990ff.png)![image](https://user-images.githubusercontent.com/622217/141329488-1e05e2e8-40f0-485c-8b40-9fe241505bc8.png)

  # To Test

1. Be disconnected
2. Open the page `Profile`
2. reduce the screen width down to 320th px
3. see the connect a wallet btn text cropped
